packages/react/.storybook/webpack.config.js

```diff
@@ -14,13 +14,12 @@ const customProperties = require('postcss-custom-properties');
 const {
   CARBON_REACT_STORYBOOK_USE_CUSTOM_PROPERTIES = 'false',
   CARBON_REACT_STORYBOOK_USE_EXTERNAL_CSS,
-  CARBON_REACT_STORYBOOK_USE_STYLE_SOURCEMAP,
```

Comment:
if we're replacing this with `NODE_ENV` to toggle sourcemaps do we need to document this anywhere?

Reply:
Up to you! I'm down either way

Reply:
this is probably fine. I thought it was already documented, but it doesn't seem like it is